src/toggleSwitch/style.js

```diff
@@ -0,0 +1,122 @@
+var tinycolor = require("tinycolor2"); 
+
+module.exports = function createColorThemeBackground(theme) {
+
+	var colors = theme;
+
+	var factor2 = 5;
+	var factor3 = 10;
+	var factor4 = 15;
+	var factor5 = 20;
+	var factor6 = 25;
+
+	function createColorShades(color) {
+		var color1 = tinycolor(color);
```

Comment:
please remove all of the unused color variables. after that remove all of the unused factor variables, etc.